docs/source/API/containers/DynRankView.rst

```diff
@@ -70,19 +70,11 @@ Description
       are specified, ``MemorySpace`` must come before ``MemoryTraits``.
 
 
-   .. rubric:: Public Enums
+   .. rubric:: Public Static Variables
 
-   .. cppkokkos:type:: rank
-
-      Rank of the view (i.e. the dimensionality).
-
-   .. cppkokkos:type:: rank_dynamic
-
-      Number of runtime determined dimensions.
-
-   .. cppkokkos:type:: reference_type_is_lvalue_reference
-
-      Whether the reference type is a C++ lvalue reference.
+   * ``rank``: Rank of the view (i.e. the dimensionality).
+   * ``rank_dynamic``: Number of runtime determined dimensions.
+   * ``reference_type_is_lvalue_reference``: Whether the reference type is a C++ lvalue reference.
```

Comment:
comment: this was done per my suggestion to be consistent with how we document this in the View page